.github/workflows/tests.yml

```diff
@@ -132,7 +132,7 @@ jobs:
             export SET_RESOLVER="--resolver ${{ matrix.resolver }}"
           fi
 
-          for exercise in ${{ github.workspace }}/exercises/*/; do
+          for exercise in ${{ github.workspace }}/exercises/practice/; do
```

Comment:
I just tested locally that: `for exercise in ${{ github.workspace }}/exercises/practice/*; do` worked for me (that `*` is needed for bash expansion)